_observing-your-data/trace/ta-dashboards.md

```diff
@@ -10,16 +10,106 @@ redirect_from:
+The Trace Analytics plugin offers at-a-glance visibility into application performance based on OpenTelemetry protocol data.
+Introduced 2.15
+{: .label .label-purple }
 
-The **Dashboard** view groups traces together by HTTP method and path so that you can see the average latency, error rate, and trends associated with a particular operation. For a more focused view, try filtering by trace group name.
+OpenSearch Observability Trace Analytics offers an improved user experience (UX) with OpenTelemetry (OTel) protocol analytics, providing comprehensive distributed systems insights. The improved UX enables you to correlate signals, aiding in identifying relationships and root causes of issues.
+[OTel](https://opentelemetry.io/), an open-source observability framework, offers standardized instrumentation for collecting telemetry data from cloud-native software.
+## Trace analytics capabilities
 
-![Dashboard view]({{site.url}}{{site.baseurl}}/images/ta-dashboard.png)
+This improved trace-analytics experience enables you to analyze and visualize services, traces, and rate, error and duration (RED) metrics.
 
+- [Services](https://opentelemetry.io/docs/specs/semconv/resource/#service) represent the different components of a distributed application. They are a significant logical term used to measure and monitor the application's building blocks to validate the system's health.
+- [Traces](https://opentelemetry.io/docs/concepts/signals/traces/) provide end-to-end visibility into requests' journeys across services, offering insights into latency and performance issues.
+- [RED metrics](https://opentelemetry.io/docs/specs/otel/metrics/api/) RED metrics monitor service health and performance by focusing on requests per second (rate), failed requests (errors), and request processing time (duration). 
+
+
+
+The Trace Analytics shows multiple service-visualizations (table and map) - which allow a logical analysis of the service's behavior and correctness. 
+
+The following visualizations represent various ways that will assist the user to quickly identify anomalies and errors
+
+1) In the **Services table** each column shows one of the RED indicators (Requests, Errors, Duration ) in addition to the connected up/down stream services and other actions
+![Services Table]({{site.url}}{{site.baseurl}}/images/trace-analytics/services-table.png)
+
+1.2) The Service table allows filtering the shown services using a general purpose filter selection which supports any field to use as a filter or a filter composition. 
+![Services Filter selection]({{site.url}}{{site.baseurl}}/images/trace-analytics/services-filter-selection.png)
+
+1.3) The Service throughput tooltip shows a service's throughput in a glans to allow quick and simple analysis of the amount of incoming requests during the past 24 hours
```

Comment:
minor. should be `glance`

Reply:
Doc is rewritten, so this feedback is resolved.